checks.d/php_fpm.py

```diff
@@ -0,0 +1,120 @@
+# 3p
+import requests
+
+# project
+from checks import AgentCheck
+from util import headers
+
+
+class PHPFPMCheck(AgentCheck):
+    """
+    Tracks basic php-fpm metrics via the status module
+    Requires php-fpm pools to have the status option.
+    See http://www.php.net/manual/de/install.fpm.configuration.php#pm.status-path for more details
+    """
+
+    SERVICE_CHECK_NAME = 'php_fpm.alive'
+
+
+    GAUGES = {
+        'listen queue': 'php_fpm.listen_queue',
+        'max listen queue': 'php_fpm.max_listen_queue',
+        'listen queue len': 'php_fpm.listen_queue_len',
+        'idle processes': 'php_fpm.idle_processes',
+        'active processes': 'php_fpm.active_processes',
+        'total processes': 'php_fpm.total_processes',
+        'max active processes': 'php_fpm.max_active_processes',
+        'max children reached': 'php_fpm.max_children_reached'
+    }
+    COUNTERS = {
+        'accepted conn': 'php_fpm.accepted_conn',
+        'slow requests': 'php_fpm.slow_requests'
+    }
+
```

Comment:
* good idea
* will try that, after re-reading the doc
* I am not sure about this one, the integration is named **PHP-FPM**, I feel like we should try to keep some consistency in names, or change everything from `php_fpm` to `fpm` then but I find it confusing (other stuff is called fpm too...)
* I think you meant `can_connect`, in this case
    * `can_connect` is correct in a way, but it's an understatement, because we got a pong from php fpm, meaning the server is ready to process requests
    * `is_running` sound a little bit like a process check to me
    * it's closer to an "haproxy-like" status heartbeat (people actually use it for that), I am open to suggestions, what do you think?